scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -4,6 +4,17 @@ scaffolding_load() {
   _setup_funcs
   _setup_vars
 
+# /src is the default value for $SRC_PATH
+# if it is set to something else
+# this scaffolding will not work
+if [ "$SRC_PATH" = "/src" ]
+then
+    continue
+else
+    e="Please do not use pkg_source in your plan when using the ruby scaffolding."
```

Comment:
I think that's a very fine warning. If we can ever figure out how to support this, this warning goes away and happiness results!